gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/Value.java

```diff
@@ -80,9 +80,9 @@ public final B fromProto(DatastoreV1.Value proto) {
       return builder.build();
     }
 
-    protected abstract V getValue(DatastoreV1.Value from);
```

Comment:
Because Marshallers implementations are package scope I don't think it has an impact on the Javadoc.